scss/_nav.scss

```diff
@@ -91,14 +95,16 @@
 //
 
 .nav-fill {
-  .nav-item {
+  .nav-item,
+  .nav-link {
     flex: 1 1 auto;
     text-align: center;
   }
 }
 
 .nav-justified {
-  .nav-item {
+  .nav-item,
+  .nav-link {
```

Comment:
I fixed this issue by requiring `.nav-item` for fill and justified variants. You can drop the selector changes here.